src/sentry/tasks/reports.py

```diff
@@ -564,7 +564,10 @@ def fetch(self, timestamp, duration, organization, projects):
 backend = RedisReportBackend(redis.clusters.get("default"), 60 * 60 * 3)
 
 
-@instrumented_task(name="sentry.tasks.reports.prepare_reports", queue="reports.prepare")
+@retry()
```

Comment:
Honestly I'd prefer to do one at a time, but given the week long turnaround with this I thought I'd just go ham. This article suggests it's okay to do: https://docs.celeryproject.org/en/stable/faq.html#faq-acks-late-vs-retry

Reply:
You should include a `max_retries` for the `instrumented_task` decorator or celery will always retry which can amplify the impacts of an outage as tasks will fail, respawn and then fail in a loop. We often use `5` as our retry limit.